cloud_service/upgrading-cloud/upgrade-cloudsvc-roxctl.adoc

```diff
@@ -0,0 +1,68 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="upgrade-cloudsvc-roxctl"]
+= Manually upgrading secured clusters in RHACS Cloud Service by using the `roxctl` CLI
+include::modules/common-attributes.adoc[]
+:context: upgrade-cloudsvc-roxctl
+
+toc::[]
+
+[role="_abstract"]
+You can upgrade your secured clusters in {product-title-managed-short} by using the `roxctl` CLI.
+
+[IMPORTANT]
+====
+You need to perform the manual upgrade procedure only if you used the `roxctl` CLI to install {product-title-managed-short}.
```

Comment:
This renders as
>  You need to perform the manual upgrade procedure only if you used the roxctl CLI to install RHACS Cloud Service.

which I think should be something like `to install the secured cluster(s)`